src/components/cards/SpeakerCard.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import {Image , View , TouchableOpacity , Text, StyleSheet, ImageSourcePropType} from "react-native"
+import { colors } from "../../constants/Colors";
+import { fonts } from '../../assets/fonts/fonts';
+
+export interface SpeakerCardProps {
+    id: string;
+    ProfilePicture : ImageSourcePropType;
+    SpeakersName : String ;
+    Content : String;
+}
+
+export default function (props : SpeakerCardProps) : JSX.Element {
+    return (
+        <View style={styles.container}>
+            <Image source={props.ProfilePicture} style={styles.image} />
+            <Text style={styles.title}>{props.SpeakersName}</Text>
+            <Text style={styles.content}>{props.Content}</Text>
+            <TouchableOpacity style={styles.button}>
+                <Text style={styles.buttontext}>
+                    SESSION
+                </Text>
+            </TouchableOpacity>
+        </View>
+    )
+}
+
+const styles = StyleSheet.create({
+    container: {
+        flex : 1 ,
+        backgroundColor : colors.DROIDCONKE_PEARL,
+        alignItems : "center",
+        borderRadius : 8,
+        margin : 5
+    },
+    image : {
+        borderColor : colors.DROIDCONKE_GREEN,
+        borderWidth : 2,
+        height : 109,
+        width : 109,
+        margin : 20,
+        borderRadius : 8
```

Comment:
The image on the design should have border-radius of 12px as per the design [https://xd.adobe.com/view/dd5d0245-b92b-4678-9d4a-48b3a6f48191-880e/screen/ff847474-c6b6-44a2-bb46-0acdffa33f9a/specs/](https://xd.adobe.com/view/dd5d0245-b92b-4678-9d4a-48b3a6f48191-880e/screen/ff847474-c6b6-44a2-bb46-0acdffa33f9a/specs/).

**Note**
- I guess the design guys changed some design properties halfway this time. That's why maybe you had border-radius as 8

Reply:
Let me check the designs and change as expected